participating.md

```diff
@@ -0,0 +1,145 @@
+# Participating in the CircuitPython weekly meeting
+
+Anyone is welcome to participate in the CircuitPython weekly meeting. Please
+read our [Code of Conduct](https://github.com/adafruit-circuitpython-weekly-meeting/blob/master/CODE_OF_CONDUCT.md)
+before participating to help this project stay welcoming.
+
+The meeting is held in the English language and usually lasts between 60 and 90
+minutes.  If you can only attend part of the meeting, that's absolutely fine.
+
+To participate in the meeting, or just attend it live as a listener, join us on
+the [Adafruit Discord](https://adafru.it/discord). There is a _#CircuitPython_
+text channel and a _#CircuitPython_ voice channel.
+
+The text from the text channel and the audio from the voice channel are
+recorded and uploaded to YouTube and podcast services. Please keep this in
+mind if you plan to participate.
+
+Before speaking in the meeting, ask any Community Moderator to add you to the
+_@CircuitPythonistas_ role. In this role, you will receive a small number of
+notifications each week, almost always related to the meeting. You can mute
+these notifications or request to leave the role anytime you like.
+
+While we love to hear the voices of everyone in the community, we also
+understand that it doesn't always work out to speak during the
+meeting—whether you don't have a microphone, are in a noisy environment, are
+simply uncomfortable speaking, or for any other reason—so the moderator will
```

Comment:
Consider using "host" instead of "moderator". Same applies for all other uses of "moderator", with the obvious exception of the "Community Moderator" reference.

Reply:
would you believe I could not remember the word "host"?

Reply:
Entirely! 😂